internal/pkg/poster/poster.go

```diff
@@ -76,128 +70,130 @@ func (p *Poster) StopPoSting() {
 }
 
 func (p *Poster) startPoStIfNeeded(ctx context.Context, newHead block.TipSet) error {
-	p.postMutex.Lock()
-	defer p.postMutex.Unlock()
-
-	if p.postCancel != nil {
-		// already posting
-		return nil
-	}
-
-	tipsetHeight, err := newHead.Height()
-	if err != nil {
-		return err
-	}
-
-	root, err := p.chain.GetTipSetStateRoot(ctx, newHead.Key())
-	if err != nil {
-		return err
-	}
-
-	stateView := p.stateViewer.StateView(root)
-	provingPeriodStart, _, _, err := stateView.MinerProvingPeriod(ctx, p.minerAddr)
-	if err != nil {
-		return nil
-	}
-
-	if provingPeriodStart > tipsetHeight {
-		// it's not time to PoSt
-		return nil
-	}
-
-	ctx, p.postCancel = context.WithCancel(ctx)
-	go p.doPoSt(ctx, stateView, provingPeriodStart, newHead.Key())
-
+	// TODO: Winning PoSt
```

Comment:
This code is now very out of sync. It will be brought back soon as part of a windowed PoSt effort.